docs/overview/getting-started/virtual-machines.md

```diff
@@ -0,0 +1,94 @@
+# Virtual Machines
+
+Avalanche allows developers to implement their Dapp in the language of their choice using the 
```

Comment:
I don't think this quite captures the usefulness of VMs. I think you should emphasize their ability to build on existing paradigms or rewrite the rules to redefine what blockchain can be.